python/ray/rllib/train.py

```diff
@@ -104,6 +104,8 @@ def run(args, parser):
             args.experiment_name: {  # i.e. log to ~/ray_results/default
                 "run": args.run,
                 "checkpoint_freq": args.checkpoint_freq,
+                "keep_checkpoint": args.keep_checkpoint,
```

Comment:
`keep_checkpoint` naming is counterintuitive (it suggests boolean and does not imply what it is doing). Something like `keep_checkpoints_num` might be better, or something other. Same for `keep_best_checkpoint`.

Reply:
Agree, had no better idea that isn't too lengthy or wierd. Can we agree on this @nikola-j @bjg2

Reply:
checkpoints_num
checkpoints_best_num